pkg/data/components/HoneycombExporter.yaml

```diff
@@ -63,3 +118,33 @@ templates:
         suppress_if: '{{ eq "none" (or .HProps.APIKey .User.APIKey) }}'
       - key: AccessKeys.SendKeyMode
         value: "{{ .Values.Mode }}"
+  - kind: collector_config
+    name: honeycombexporter_collector
+    format: collector
+    meta:
+      componentSection: exporters
+      signalTypes: [traces, metrics, logs]
+      collectorComponentName: otlphttp
+    data:
+      - key: "{{ .ComponentName }}.endpoint"
+        value: "{{ .Values.APIEndpoint }}"
+      - key: "{{ .ComponentName }}.tls.insecure"
+        value: "{{ .HProps.Insecure | encodeAsBool }}"
+        suppress_if: "{{ not .HProps.Insecure }}"
+      - key: "{{ .ComponentName }}.headers.x-honeycomb-team"
+        value: "{{ .Values.APIKey }}"
+      - key: "{{ .ComponentName }}.headers"
+        value: "{{ .HProps.Headers | encodeAsMap }}"
+        suppress_if: "{{ not .HProps.Headers }}"
+      - key: "{{ .ComponentName }}.sending_queue.enabled"
+        value: "{{ true | encodeAsBool}}"
```

Comment:
the sending_queue is enabled by default for the otlphttp exporter, we can exclude this.